docs/advanced/pcidevices.md

```diff
@@ -64,3 +66,32 @@ Boot the VM up, and run `lspci` inside the VM, the attached PCI devices will sho
 ## Installing drivers for your PCI device inside the VM
 
 This is just like installing drivers in the host. The PCI passthrough feature will bind the host device to the `vfio-pci` driver, which gives VMs the ability to use their own drivers. [Here is a screenshot](https://tobilehman.com/posts/suse-harvester-pci/#toc) of NVIDIA drivers being installed in a VM. It includes a CUDA example that proves that the device drivers work.
+
+## SRIOV Network Devices
+_Available as of v1.2.0_
+
+![](/img/v1.2/pcidevices/SriovNetworkDevicesLink.png)
+
+The `pcidevices-controller` addon can now scan network interfaces on the underlying hosts and check if they support SRIOV Virtual Functions (VFs). If a valid device is found, `pcidevices-controller` will generate a new `SRIOVNetworkDevice` object.
+
+![](/img/v1.2/pcidevices/SriovNetworkDevicesList.png)
+
+To create VFs on a SriovNetworkDevice, you can click **⋮ > Enable** and then define the **Number of Virtual Functions**.
+![](/img/v1.2/pcidevices/SriovNetworkDeviceEnable.png)
+
+![](/img/v1.2/pcidevices/SriovNetworkVFDefinition.png)
+
+The `pcidevices-controller` will define the VFs on the network interface and report the new PCI device status for the newly created VFs.
+
+![](/img/v1.2/pcidevices/SriovNetworkDevicesVFStatus.png)
+
+On the next re-scan, the `pcidevices-controller` will create the PCIDevices for VFs. This can take up to 1 minute.
```

Comment:
```suggestion
On the next re-scan, the `pcidevices-controller` will create the PCI devices for VFs. This can take up to 1 minute.
```
Should PCIDevice be replaced by PCI device?

Reply:
actually the CRD is `PCIDevice` only so that is correct.